src/test/kotlin/no/skatteetaten/aurora/fergus/service/StorageGridServiceAuthorizeTest.kt

```diff
@@ -0,0 +1,77 @@
+package no.skatteetaten.aurora.fergus.service
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import kotlinx.coroutines.runBlocking
+import no.skatteetaten.aurora.fergus.controllers.AuthorizationPayload
+import okhttp3.mockwebserver.MockResponse
+import okhttp3.mockwebserver.MockWebServer
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.openapitools.client.model.AuthorizeResponse
+import org.openapitools.client.model.AuthorizeResponse.StatusEnum.SUCCESS
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.http.HttpHeaders.AUTHORIZATION
+import org.springframework.http.HttpHeaders.CONTENT_TYPE
+import org.springframework.http.MediaType.APPLICATION_JSON_VALUE
+
+@SpringBootTest(
+    properties = [
+        "integrations.storagegrid.url=http://localhost:9999",
+        "aurora.token.value=testToken",
+    ]
+)
+class StorageGridServiceAuthorizeTest {
+    @Autowired
+    lateinit var storageGridService: StorageGridService
+    @Autowired
+    lateinit var objectMapper: ObjectMapper
+
+    private lateinit var mockWebServer: MockWebServer
+
+    @BeforeEach
+    fun setup() {
+        mockWebServer = MockWebServer()
+        mockWebServer.start(9999)
+    }
+
+    @AfterEach
+    fun tearDown() {
+        mockWebServer.shutdown()
+    }
+
+    @Test
+    fun authorizeDeepHappyTest() {
```

Comment:
Skriv gjerne testnavn med backticks `
På den måten kan du skrive en setning, som `Authorize deep happy test`

Nå sier ikke det testnavnet så mye, kanskje det heller burde si noe om input/outputen. Hva som forventes?

Reply:
Takk for tipset.  Skal se på det.

Reply:
Happy/sunshine-tester synes jeg kanskje ikke trenger så mye forklaring. Jeg skal beskrive testene bedre for feilsituasjoner og spesialtilfeller.